backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -455,85 +464,96 @@ export class UserResolver {
       description,
       username,
       language,
-      publisherId,
       password,
       passwordNew,
       coinanimation,
     }: UpdateUserInfosArgs,
     @Ctx() context: any,
-  ): Promise<UpdateUserInfosResponse> {
-    const payload = {
-      session_id: context.sessionId,
-      update: {
-        'User.first_name': firstName || undefined,
-        'User.last_name': lastName || undefined,
-        'User.description': description || undefined,
-        'User.username': username || undefined,
-        'User.language': language || undefined,
-        'User.publisher_id': publisherId || undefined,
-        'User.password': passwordNew || undefined,
-        'User.password_old': password || undefined,
-      },
-    }
-    let response: UpdateUserInfosResponse | undefined
+  ): Promise<boolean> {
     const userRepository = getCustomRepository(UserRepository)
+    const userEntity = await userRepository.findByPubkeyHex(context.pubKey)
+    const loginUser = await LoginUser.findOneOrFail({ email: userEntity.email })
```

Comment:
Same reflexion as above never know when LoginUser.findOneOrFail or LoginUser.count works or not so I would suggest:
```suggestion
    const loginUserRepository = getCustomRepository(LoginUserRepository)
    const loginUser = await loginUserRepository.findOneOrFail({ email: userEntity.email })
```